examples/user-notification.amp.html

```diff
@@ -209,7 +209,7 @@
       data-dismiss-href="http://localhost:8000/api/echo/post">
       This site uses cookies to personalize content.
       <a href="#learn-more">Learn more.</a>
-     <button on="tap:amp-user-notification1.dismiss">I accept</button>
+     <button on="tap:amp-user-notification1.dismiss" role="button" tabindex="0">I accept</button>
```

Comment:
In my over zealousness I got redundant. This will be removed from all buttons here.